db/l2db/apiqueries.go

```diff
@@ -127,3 +127,57 @@ func (l2db *L2DB) GetTxAPI(txID common.TxID) (*PoolTxAPI, error) {
 		txID,
 	))
 }
+
+// GetPoolTxs return Txs from the pool
+func (l2db *L2DB) GetPoolTxs(fromIdx, toIdx *common.Idx, state *common.PoolL2TxState) ([]*PoolTxAPI, error) {
```

Comment:
We can have transactions without the `toIdx` into the transaction pool, like the [TransferToBjj](https://docs.hermez.io/#/developers/protocol/hermez-protocol/protocol?id=transfertobjj) transaction, this performing a `CreateAccountBjj` transaction to create the idx before executing the transaction itself, but while that the tx stays into the `tx_pool` table, it doesn't have the `toIdx`. 

The same for [TransferToEthAddr](https://docs.hermez.io/#/developers/protocol/hermez-protocol/protocol?id=transfertoethaddr) 

Maybe it's a good idea to add the `toBjj` and `toEthAddr` parameters to filter the pool's transactions.